gcloud-java-core/src/main/java/com/google/gcloud/PageImpl.java

```diff
@@ -38,6 +40,38 @@
     Page<T> nextPage();
   }
 
+  /**
+   * Base class for page fetchers.
+   *
+   * @param <T> the value type that the page holds. {@code T} should implement
+   *     {@code java.io.Serializable}.
+   * @param <ServiceOptionsT> the service options type.
+   */
+  @SuppressWarnings("rawtypes")
```

Comment:
Eclipse gives me a warning that `ServiceOptions` (in the method signature) is a raw type, so I added it in.